test/data/azure/create_triton.py

```diff
@@ -0,0 +1,78 @@
+from onnx import helper, TensorProto, save
+# from onnxruntime import *
+
+def createAddf():
+    auth_token = helper.make_tensor_value_info('auth_token', TensorProto.STRING, [-1])
+    X = helper.make_tensor_value_info('X', TensorProto.FLOAT, [-1])
+    Y = helper.make_tensor_value_info('Y', TensorProto.FLOAT, [-1])
+    Z = helper.make_tensor_value_info('Z', TensorProto.FLOAT, [-1])
+    invoker = helper.make_node('AzureTritonInvoker', ['auth_token', 'X', 'Y'], ['Z'],
+                               domain='com.microsoft.extensions', name='triton_invoker',
+                               model_uri='https://endpoint-5095584.westus2.inference.ml.azure.com',
+                               model_name='addf', model_version='1', verbose='1')
+    graph = helper.make_graph([invoker], 'graph', [auth_token, X, Y], [Z])
+    model = helper.make_model(graph,
```

Comment:
will this make_model function cause the IR version issue?

Reply:
Not sure what the issue is. The script is just to re-create the checked in models if/when needed. It's not run as part of any CI..